### ISSUE TITLE
Memes

### ISSUE THREAD

**Issue by l-2-j** [neos-modding-group/world#4]
There's a lack of memes in the world, we at least need to hide
![the seal](https://user-images.githubusercontent.com/5854483/185755984-5e8892b0-546e-4cf6-bb29-79bafb9aafad.jpg) 
and the original version too somewhere

**Comment by rampa3** [comment]
Yeah, some NMG meme easter eggs would be nice. But definitely should not go overboard with them - it is mod hub, not meme world.
